hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/WriteStatus.java

```diff
@@ -181,6 +188,11 @@ private void updateStatsForFailure() {
     totalErrorRecords++;
   }
 
+  public void removeMetadataStats() {
```

Comment:
didn't we also agree to drop failed records info as well. 
we might need another api for that.

Reply:
Addressed